### PR TITLE
Feature/closecontact

### PR DESCRIPTION
**Pull request by khihadysucahyo**
## Init revamped feature/closecontact

### CODE REVIEW

File: services/close_contact.v2.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  try {
+    const results = await Case.find({
+      status: CRITERIA.CLOSE,
+      close_contact_premier: {
+        $elemMatch: {
+          close_contact_id_case: pre.id_case
+        }
+      },
+      delete_status: { $ne: 'deleted' }
+    }).populate(['author'])
```
Comment (firmanJS) [inline services/close_contact.v2.js:22]: ini gak akan di ganti wkwkwk ?

**Review by firmanJS — APPROVED** [review]

File: services/close_contact.v2.js (hunk 1 above)
Comment (khihadysucahyo) [reply services/close_contact.v2.js]: wkkw diganti dong kang

**Comment by khihadysucahyo** [comment]
@firmanJS udah diubah jadi sobat aggregate kang wkwk

**Comment by firmanJS** [comment]
> @firmanJS udah diubah jadi sobat aggregate kang wkwk

mantapss

**Review by sandisunandar99 — APPROVED** [review]

**Pull request merged**